radboudumc-template/templates/new/publication-thesis.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %}
```

Comment:
General comment regarding thesis pages: they are not generated for me. Please check if this works in the parser.

Reply:
this is fixed